### PR TITLE
Created basic grid for the category page (#1xxujqt)

### PR DESCRIPTION
**Pull request by azkyakhan**
127 0 0 1_5500_pages_category html (5)

127 0 0 1_5500_pages_category html (4)

### CODE REVIEW

File: pages/category.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      <h1>HOTWAX COMMERCE</h1>
+      <h2>Omnichannel Order Management System for Retail Brands</h2>
+      <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+    </div>
+
+    <!-- Shopify apps cards-->
+    <div>
+      <h2 class="card-header">Shopify apps</h2>
+
+      <div class="cards-container">
```
Comment (dt2patel) [inline pages/category.html]: rename class to "apps"

File: pages/category.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        <div class="cards">
```
Comment (dt2patel) [inline pages/category.html]: rename to "card" and add class "app"

File: pages/category.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          <div>
```
Comment (dt2patel) [inline pages/category.html:38]: do we need this extra div?

File: pages/category.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+  <body>
+    <!-- Logo-->
+    <div>
+      <a href="">
+        <img src="../img/HC Logo.svg" class="logo" alt="Logo">
+      </a>
+    </div>
+
+    <div class="banner-content">
```
Comment (dt2patel) [inline pages/category.html]: use header tag here

File: pages/category.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.banner-content,
+.card-header {
+  text-align: center;
+}
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(275px, 1fr));
```
Comment (dt2patel) [inline pages/category.html]: this grid will be for desktop, on mobile we want the cards to be just one after another vertically similar to the concept we use in the BOPIS fulfillment app. optimize this styling to be mobile first.

File: pages/category.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  gap: 10px;
+  box-sizing: border-box;
```
Comment (dt2patel) [inline pages/category.html]: align text to center

File: pages/category.html [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.cards {
```
Comment (dt2patel) [inline pages/category.html]: manage card styling globally, we may want to use this later too. on this page app cards should have two classes, app and card. card will be styled globally and we will use app class to manage style here semantically.

Comment (dt2patel) [reply pages/category.html]: For card stying reference the cards at the bottom of the current home page of Hotwax.co

File: pages/category.html [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+    </div>
+
+    <!-- Shopify apps cards-->
+    <div>
+      <h2 class="card-header">Shopify apps</h2>
+
+      <div class="cards-container">
+        <div class="cards">
+          <div>
+            <a href="">
```
Comment (dt2patel) [inline pages/category.html]: use the anchor tag to wrap the div card. it should be the top most tag

File: pages/category.html [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+    </div>
+
+    <div class="banner-content">
+      <h1>HOTWAX COMMERCE</h1>
+      <h2>Omnichannel Order Management System for Retail Brands</h2>
+      <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+    </div>
+
+    <!-- Shopify apps cards-->
+    <div>
```
Comment (dt2patel) [inline pages/category.html]: use nav tag

**Review by dt2patel — CHANGES_REQUESTED** [review]
Needs work

File: pages/category.html (hunk 1 above)
Comment (azkyakhan) [reply pages/category.html]: Done

File: pages/category.html (hunk 2 above)
Comment (azkyakhan) [reply pages/category.html]: Done

File: pages/category.html (hunk 3 above)
Comment (azkyakhan) [reply pages/category.html:38]: Removed sir

File: pages/category.html (hunk 4 above)
Comment (azkyakhan) [reply pages/category.html]: Done sir

File: pages/category.html (hunk 5 above)
Comment (azkyakhan) [reply pages/category.html]: Done

File: pages/category.html (hunk 6 above)
Comment (azkyakhan) [reply pages/category.html]: Done

File: pages/category.html (hunk 7 above)
Comment (azkyakhan) [reply pages/category.html]: Done

File: pages/category.html (hunk 8 above)
Comment (azkyakhan) [reply pages/category.html]: Done

File: pages/category.html (hunk 9 above)
Comment (azkyakhan) [reply pages/category.html]: Done

**Comment by azkyakhan** [comment]
The branch in this PR was created from `main`, that is why when I retargeted it automatically get closed. I have done the requetsed changes in the new PR :
https://github.com/hotwax/omnichannelcto-site/pull/5

**Pull request closed without merging**